instrumentation/armeria/armeria-grpc-1.14/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/armeria/grpc/v1_14/ArmeriaGrpcTest.java

```diff
@@ -85,6 +91,8 @@ void grpcInstrumentation() {
                             equalTo(RPC_GRPC_STATUS_CODE, (long) Status.Code.OK.value()),
                             equalTo(SERVER_ADDRESS, "127.0.0.1"),
                             equalTo(SERVER_PORT, (long) server.httpPort()))
+                            equalTo(RPC_RESPONSE_BODY_SIZE, responseSerializedSize),
+                            equalTo(RPC_REQUEST_BODY_SIZE, requestSerializedSize))
```

Comment:
I think all your tests are failing because this isn't formatted correctly

```suggestion
                            equalTo(SERVER_PORT, (long) server.httpPort()),
                            equalTo(RPC_RESPONSE_BODY_SIZE, responseSerializedSize),
                            equalTo(RPC_REQUEST_BODY_SIZE, requestSerializedSize))
```